### ISSUE TITLE
Webpack build error

### ISSUE THREAD

**Issue by chinhnguyen90** [niklasvh/html2canvas#2706]
I install newest version, when i start pj with webpack, source-map-loader error same below, please help me fix this error.
Thanks!

```
WARNING in ./node_modules/html2canvas-copy/dist/html2canvas.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/home/chinh/js/his_acexis_frontend/node_modules/html2canvas-copy/node_modules/text-segmentation/node_modules/src/Trie.ts' file: Error: ENOENT: no such file or directory, open '/home/chinh/js/his_acexis_frontend/node_modules/html2canvas-copy/node_modules/text-segmentation/node_modules/src/Trie.ts'

WARNING in ./node_modules/html2canvas-copy/dist/html2canvas.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/home/chinh/js/his_acexis_frontend/node_modules/html2canvas-copy/node_modules/text-segmentation/node_modules/src/Util.ts' file: Error: ENOENT: no such file or directory, open '/home/chinh/js/his_acexis_frontend/node_modules/html2canvas-copy/node_modules/text-segmentation/node_modules/src/Util.ts'

WARNING in ./node_modules/html2canvas-copy/dist/html2canvas.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/home/chinh/js/his_acexis_frontend/node_modules/html2canvas-copy/node_modules/text-segmentation/node_modules/utrie/node_modules/src/index.ts' file: Error: ENOENT: no such file or directory, open '/home/chinh/js/his_acexis_frontend/node_modules/html2canvas-copy/node_modules/text-segmentation/node_modules/utrie/node_modules/src/index.ts'
```

**Comment by unional** [comment]
This is likely cause by the folder level difference between JS code and types:

```
  "main": "dist/html2canvas.js",
  "module": "dist/html2canvas.esm.js",
  "typings": "dist/types/index.d.ts",
  "browser": "dist/html2canvas.js",
  ```

**Comment by FeiFanLiang** [comment]
me too

**Comment by mariush2** [comment]
I have this exact same issues aswell

**Comment by abetoots** [comment]
issue still persists, using create-react-app v5
`WARNING in ./node_modules/html2canvas/dist/html2canvas.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/home/ubuntu/CODE/websand2/node_modules/src/render/vector.ts' file: Error: ENOENT: no such file or directory, open '/home/ubuntu/CODE/websand2/node_modules/src/render/vector.ts'
 @ ./src/shared/utils/useImgFromHtmlEffect.js 7:0-38 27:29-40
 @ ./src/pages/authenticated/admin-template-email/emailTemplates.js 24:0-78 66:6-26 609:51-71
 @ ./src/routes.js 23:0-95 188:19-41
 @ ./src/index.js 7:0-30 29:35-41`

**Comment by niklasvh** [comment]
Fixed in 1.4.1
